sdk/test/common/random_fork_test.cc

```diff
@@ -0,0 +1,52 @@
+#ifdef __unix__
+// Verifies that IDs don't clash after forking the process.
+//
+// See https://github.com/opentracing-contrib/nginx-opentracing/issues/52
+#  include "src/common/random.h"
```

Comment:
Curious - any reason for this weird spacing between `#` and `include`?

Reply:
It's from clang-format. But, I think, because it's nested in a #if